docs/userguide/configuration.rst

```diff
@@ -1057,6 +1057,33 @@ Example configuration
 
 .. _conf-riak-result-backend:
 
+.. setting:: elasticsearch_retry_on_timeout
```

Comment:
This documentation change belongs to line 1057 (above `.. conf-riak-result-backend:`).